### DISCUSSION TITLE
How do I package only the necessary icons when building?

### DISCUSSION THREAD

**Discussion by Reverier-Xu** [Remix-Design/RemixIcon#540] [Q&A]
Hi, thank you for your awesome work!

RemixIcon is a very large icon package and I would like to be able to pack only some of the icons that been used when building.

I noticed that it is possible to bundle selected icons into a webfont file, but due to server bandwidth limitations, I would like to be able to load only the icons used on one page, rather than all of the icons used throughout the site.

Even if I use packing the necessary icons to webfont to reduce the build size, frequently checking off new icons and re-downloading packed fonts during development leads to a poor development experience.

Any suggestions other than copying the svg manually (which is very difficult to manage)?

**Comment by xiaochunjimmy** [comment]
On our [website](https://remixicon.com), click the `+` button in the upper right corner of the icons you'd to use, then the icon will be added to the [Collection](https://remixicon.com/collection). Then you can export the SVG or font files just with icons you added from the Collection.

**Reply by Reverier-Xu** [reply]
> Even if I use packing the necessary icons to webfont to reduce the build size, frequently checking off new icons and re-downloading packed fonts during development leads to a poor development experience.

It will be troublesome to maintain in this way...

**Reply by mwarnerdotme** [reply]
If you're using React to build your application, check out this discussion https://github.com/Remix-Design/RemixIcon/discussions/396 😄
